fix: addresses from env file (#453)
mangrovedao/mangrove-core: README.md

$ yarn build
```

## Addresses

> When writing scripts that uses the `Generic.sol` script, you can control what addresses are read As default it will try and look into the `{projectRoot}/addresses` folder, this can be disabled by setting `READ_ROOT_ADDRESSES` to false. If you want to read addresses from a different folder, you can set `READ_ADDRESSES_FROM` to the path of the folder you want to read from, the variable is a JSON string, here is an example:

Reviewer: I'd suggest giving the env var a prefix (eg `MGV_`) so we don't pollute the env var namespace.

Also, it looks like it's possible to point to multiple paths, so I suggest tweaking this text a bit:

```suggestion
When writing scripts that uses the `Generic.sol` script, you can control what addresses are read.

By default, it will try and look into the `{projectRoot}/addresses` folder, this can be disabled by setting `READ_ROOT_ADDRESSES` to false.

If you want to read addresses from other folders, you can set `READ_ADDRESSES_FROM` to the paths addresses should be read from. The variable is a JSON string, here is an example:
```

```shell
export MGV_ADDRESSES_PATHS='{ "paths": ["/addresses/"] }'

> export READ_ROOT_ADDRESSES=false

Reviewer: I see there's partially a `MGV_` prefix here :-)

Author: Fixed

```

> In this example we disable the default path and set the path to `/addresses/`. Remember the path is relativ to the project root. This way you can easily read addresses from multiple sources.

Reviewer: ```suggestion
In this example we disable the default path and set the path to `/addresses/`. Remember the path is relative to the project root. This way you can easily read addresses from multiple sources.
```
